.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
-dist/
```

Comment:
@claudia-afonso-credimi questo è voluto o è rimasto da alcune prove?

Reply:
@lordgordon su jfrog ho visto che il pacchettino non ha i files buildati quindi ho lanciato in locale la build aggiunto i files così

Reply:
ok